blocks/extensions/__init__.py

```diff
@@ -128,6 +128,24 @@ def on_interrupt(self):
         pass
 
 
+class CallbackName(str):
```

Comment:
The do methods if I remember correctly (on my phone, so hard to check)
On Mar 8, 2015 11:38 AM, "Dmitry Bogdanov" <notifications@github.com> wrote:

> In blocks/extensions/__init__.py
> <https://github.com/bartvm/blocks/pull/433#discussion_r26006117>:
>
> > @@ -128,6 +128,24 @@ def on_interrupt(self):
> >          pass
> >
> >
> > +class CallbackName(str):
>
> But here
> <https://github.com/bartvm/blocks/blob/master/blocks/extensions/__init__.py#L59>
> it is already callback_name. Which location in the code are you talking
> about?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bartvm/blocks/pull/433/files#r26006117>.
>